apps/daimo-mobile/src/view/screen/OnboardingScreen.tsx

```diff
@@ -228,6 +260,130 @@ function PageBubble({ count, index }: { count: number; index: number }) {
   return <View style={{ flexDirection: "row" }}>{bubbles}</View>;
 }
 
+function TryEnclave({
+  onNext,
+  createStatus,
+  forceWeakerKeys,
+  setForceWeakerKeys,
+}: {
+  onNext: () => void;
+  createStatus: ActStatus;
+  forceWeakerKeys: boolean;
+  setForceWeakerKeys: (force: boolean) => void;
+}) {
+  const enclaveKeyName = defaultEnclaveKeyName;
+  const [loading, setLoading] = useState(false);
+  const [askWeak, setAskWeak] = useState(false);
+  const [error, setError] = useState("");
+
+  // If we've failed in loading or creating the key already without force even
+  // before getting to this screen, force weaker keys now
+  useEffect(() => {
+    if (createStatus === "error" && !forceWeakerKeys) setForceWeakerKeys(true);
+  }, [createStatus]);
+
+  const trySignatureGeneration = async (tryWeaker: boolean) => {
+    setLoading(true);
+    setForceWeakerKeys(tryWeaker);
+    try {
+      await requestEnclaveSignature(
+        enclaveKeyName,
+        "dead",
+        "Create key",
+        tryWeaker
+      );
+
+      console.log(`[ONBOARDING] enclave signature trial success`);
+      onNext();
+    } catch (e: any) {
+      console.error(e);
+      if (!tryWeaker) {
+        console.error(
+          `[ONBOARDING] enclave signature trial error, asking to downgrade to weaker keys`
+        );
+        setAskWeak(true);
+      } else {
+        const err = `[ONBOARDING] Failed with weaker keys too, giving up`;
+        setError(err);
+        console.error(err);
+      }
+    }
+    setLoading(false);
+  };
+
+  return (
+    <View style={styles.onboardingScreen}>
+      <View style={styles.createAccountPage}>
+        <TextH1>
+          <Octicons name="unlock" size={40} color={askWeak ? "red" : "black"} />
```

Comment:
recommend following instead:

```tsx
<Octicons name={askWeak ? "unlock" : "lock"} size={32} />
```

Reply:
done; changed size to 40 to keep it consistent with "bell" size on notifications screen.